spec/models/post_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it 'is not valid without a title' do
+    post = Post.new
+    expect(post).to_not be_valid
+  end
+end
```

Comment:
Nice job so far. However, as part of the project requirements, any custom methods in the models should have a corresponding unit test. In view of this kindly add unit test for the `update_user_post_counter` and `five_most_recent_comments` methods in the post model here. :wink: